python/lsst/obs/decam/_instrument.py

```diff
@@ -51,6 +52,11 @@ def getName(cls):
 
     def getCamera(self):
         path = os.path.join(getPackageDir("obs_decam"), self.policyName, "camGeom")
+        return self._getCameraFromPath(path)
+
+    @classmethod
```

Comment:
Why does this need to be a class method if `getCamera` is not?

Reply:
It doesn't rely on class properties or instance properties (which is why it's a good candidate for caching) but you are right it probably should be a `@staticmethod` instead.